File: 1827-minimum-operations-to-make-the-array-increasing/1827-minimum-operations-to-make-the-array-increasing.py
class Solution:
    def minOperations(self, nums: List[int]) -> int:
        total = 0
        for i in range(len(nums)-1): # 0 1 2 3 
            if nums[i] >= nums[i+1]:
                total += nums[i] - nums[i+1] + 1
                nums[i+1] = nums[i] + 1
        return total

File: 1827-minimum-operations-to-make-the-array-increasing/1827-minimum-operations-to-make-the-array-increasing.java
class Solution {
    public int minOperations(int[] nums) {
        int total = 0;
        for (int i = 0; i < nums.length-1;i++){
            if (nums[i] >= nums[i+1]){
                total += nums[i] - nums[i+1] + 1;
                nums[i+1] = nums[i] + 1;
            }
        }
        return total;   
    }
}